components/account/AccountHeader.vue

```diff
@@ -85,7 +88,7 @@ const isNotifiedOnPost = $computed(() => !!relationship?.notifying)
 
 <template>
   <div flex flex-col>
-    <button border="b base" z-1>
+    <button border="b base" z-1 :class="!hasHeader && 'cursor-auto'">
       <img h-50 height="200" w-full object-cover :src="account.header" :alt="t('account.profile_description', [account.username])" @click="previewHeader">
     </button>
```

Comment:
Probably better not to output a button at all if it's not clickable.

```suggestion
    <component :is="hasHeader ? 'button' : 'div'" border="b base" z-1>
      <img h-50 height="200" w-full object-cover :src="account.header" :alt="t('account.profile_description', [account.username])" @click="previewHeader">
    </component>
```

Reply:
What about using a disabled button here?

Reply:
I think that would also be a good solution - do you think it has any benefits over a div in this case?

Reply:
When using a disabled button, the accessibility semantics are: there was a button here, but now it's disabled; with a div it doesn't have any semantics.

In addition, when using the disabled button to encounter an update, in Vue's diff algorithm, only update the attribute instead of creating a new DOM node

Reply:
Yes, that's true - but I'm not sure whether that's _better_ when it comes to the header of a page. Probably it is? I don't expect there will ever be any update to `hasHeader`.

Reply:
I checked where this component is used. It seems that there will be no state update, so naturally there will be no such problem.

Sorry, I only looked at this component and did not pay special attention to the page it uses.